website/docs/proxy/using-a-proxy-on-linux.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 4
+title: Linux
+description: Using Podman Desktop behind a proxy on Linux.
+tags: [podman-desktop, proxy, linux]
+keywords: [podman desktop, containers, podman, proxy, linux]
+---
+
+# Using Podman Desktop behind a proxy on Linux
+
+You can configure Podman to run behind a proxy.
+
+#### Prerequisites
+
+* `<proxy_ca.pem>`: Your proxy Certificate Authorities (CA), in Privacy-Enhanced Mail (PEM) format.
```

Comment:
This guide does not include PD at all. I am puzzled as of why we cannot use PD on linux to actually use proxy? But I did not explore the area thoroughly, though.

Reply:
PD is injecting the configuration to the VM, but not handling local files.